src/matter/cluster/AdminCommissioningCluster.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { TlvByteString, TlvField, TlvObject, TlvUInt16, TlvUInt32 } from "@project-chip/matter.js";
+import { Cluster, Command, TlvNoArguments, TlvNoResponse, OptionalCommand } from "./Cluster";
+
+export const AdminCommissioningCluster = Cluster({
+    id: 0x3c,
+    name: "AdministratorCommissioning",
+    revision: 1,
+
```

Comment:
The cluster attributes are missing, maybe leave a "TODO" to be added later? maybe reference to 11.18.7

Reply:
Done